src/aspire/utils/rotation.py

```diff
@@ -72,12 +72,10 @@ def angles(self):
         """
         rotations = sp_rot.from_matrix(self._matrices.astype(self.dtype))
 
-        # If `about_axis` was used to generate rotations about the z-axis
-        # turn off Gimbal lock warning.
+        # Filter Gimbal lock warnings.
         with warnings.catch_warnings():
-            if self._about_z:
-                msg = "Gimbal lock detected*"
-                warnings.filterwarnings("ignore", message=msg, category=UserWarning)
+            msg = "Gimbal lock detected*"
+            warnings.filterwarnings("ignore", message=msg, category=UserWarning)
```

Comment:
Do we never want to be warned about gimbal lock? If so, I'm fine with this. If not, we could just change the updated rotations in the test.

Reply:
I think they are going to keep occurring whenever we do trivial about axis based tests... All we did before was carve out the default axis to ignore.

Reply:
Let's keep it like this.

Reply:
lol, too late I'm testing the changes. More code to review :D

Reply:
Lol, you're quick! I'll take a look.

Reply:
Updated.

One of the test I avoid the gimble lock (most of the time I guess) by using a random rotation.

For the others there is an option to disable now.